src/main/java/seedu/commands/CommandUtils.java

```diff
@@ -36,4 +31,13 @@ public static LocalDate formatDate(String stringDate, String commandFormat) {
             throw new WiagiInvalidInputException(INCORRECT_DATE_FORMAT + commandFormat);
         }
     }
+
+    public static double roundAmount(String stringAmount, String commandFormat) {
```

Comment:
if this method is only used for budget, should we change it to roundBudget?

Reply:
it is used inside command utils as well